tests/test_basic_read.csvy

```diff
@@ -0,0 +1,9 @@
+# ---
+title: Sample Dataset
+author: User
+description: "This is a test CSVY file"
+# ---

```

Comment:
This might work, but it's not what would be expected and might fail in other readers. The comment character needs to be in every line:
```suggestion
# ---
# title: Sample Dataset
# author: User
# description: "This is a test CSVY file"
# ---

```